assets/js/components/OptIn.js

```diff
@@ -72,17 +72,22 @@ export default function OptIn( { id, name, className, optinAction } ) {
 				disabled={ saving }
 				onChange={ handleOptIn }
 			>
-				{ createInterpolateElement(
-					__( 'Help us improve the Site Kit plugin by allowing tracking of anonymous usage stats. All data are treated in accordance with <a>Google Privacy Policy</a>', 'google-site-kit' ),
-					{
-						a: <Link
-							key="link"
-							href={ 'https://policies.google.com/privacy' }
-							external
-							inherit
-						/>,
+				<span>Help us improve Site Kit by sharing anonymous usage data.</span>
+				<br />
+				<span>
+					{ createInterpolateElement(
+						__( 'All collected data is treated in accordance with the <a>Google Privacy Policy</a>', 'google-site-kit' ),
```

Comment:
Breaking out the second sentence onto a new line makes it look slightly better on wide screens, but worse on medium and small screens. Let's do it only for screens with a size that is bigger than the desktop breakpoint and display these sentences in a single line on smaller screens. Something like this:

```css
.googlesitekit-opt-in span {
    @media (min-width: $bp-desktop) {
        display: block;
    }
}
```

Reply:
Excellent point! There were no designs for different screen sizes. I should have tested this @eugene-manuilov  :smile: